anago

```diff
@@ -565,8 +565,13 @@ checkout_object () {
   fi
 
   # Checkout location
-  logecho -n "Checking out $tree_object: "
-  logrun -s git checkout $branch_arg $tree_object $branch_point || return 1
+  local checkout_args=()
```

Comment:
Add a comment above this giving guidance on how `checkout_args`, `branch_arg`, and `tree_object` should be invoked.
Prefer explicit language here instead of "maybe" or "hopefully".

Reply:
Those are not invoked ... but anyway, I hope the comment I added clears things up a bit.